libraries/node-core-library/src/FileSystem.ts

```diff
@@ -904,17 +900,17 @@ export class FileSystem {
       // since writev() doesn't take an argument for where to start writing.
       const toCopy: Uint8Array[] = [...contents];
```

Comment:
Can we generalize this API to accept the same argument type as `writev` accepts? It's a bit broader than `Uint8Array`.